### PR TITLE
GEOPY-1413: Move all optional parameters to general tabs

### PR DESCRIPTION
**Pull request by benk-mira**
**GEOPY-1413 - Move all optional parameters to general tabs**
…og file permissions all of a sudden.

### CODE REVIEW

**Review by domfournier — COMMENTED** [review]
Looks good. Can you also change the toml file to use the new alpha format for geoh5py? I ll need it on GEOPY-1423

File: las_geoh5_assets/uijson/import_las_files.ui.json [hunk 1]
```diff
@@ -4,27 +4,14 @@
...
+        "label": "Run python module ",
+        "tooltip": "Warning: launches process to run python model on save",
+        "main": true
     },
     "monitoring_directory": "",
-    "conda_environment": "las-geoh5",
+    "conda_environment": "",
     "conda_environment_boolean": false,
     "workspace": "",
-    "drillhole_group": {
```
Comment (domfournier) [inline las_geoh5_assets/uijson/import_las_files.ui.json:15]: I think you might have deleted too much. Getting a crash with this missing
image

**Review by domfournier — COMMENTED** [review]
Check missing value

File: las_geoh5_assets/uijson/import_las_files.ui.json (hunk 1 above)
Comment (benk-mira) [reply las_geoh5_assets/uijson/import_las_files.ui.json]: yep, you're right.. whoops!

**Review by domfournier — APPROVED** [review]
Looks good, and tested

**Pull request merged**